components/tidb_query/src/expr/builtin_math.rs

```diff
@@ -157,13 +156,13 @@ impl ScalarFunc {
     pub fn rand(&self, _: &mut EvalContext, _: &[Datum]) -> Result<Option<f64>> {
         let mut cus_rng = self.cus_rng.rng.borrow_mut();
```

Comment:
Could you change this part as well? `self.cus_rng` is no longer needed

Reply:
refactored.